App/localization/languages/es/screen_home.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Would it be possible to have one file per language? It would make things easier imo, and help with integration with external tools (e.g. POEditor).

Reply:
The currently extracting tool doesn't seem to allow me doing that. But I can write a script to merge all of them into one. Gimme a day (I'm not very familliar with structure problem, but I can do it)

Reply:
>  (btw, it may kill the purpose of load namespace performance)

If it's a trade-off between performance and readability, I prefer readability, so would really like to have 1 file per language. Performance is really not an issue for this app.